js/packages/cli/src/commands/mint-nft.ts

```diff
@@ -75,7 +76,7 @@ export const createMetadata = async (
       new Creator({
         address: creator.address,
         share: creator.share,
-        verified: 1,
+        verified: verifyCreators ? 1 : 0,
```

Comment:
@austbot assuming `verifyCreators` is `true` and there are creators other than the one running the command, this will fail trying to verify ALL of them at once, right? The program won't allow the creator running the command to sign/verify the other creators.

Shouldn't the `verifyCreators` (and `--no-verify-creators` option) be changed to singular and then this line be changed to something like the following?

(untested)
```suggestion
        verified: (creator.address = wallet.publicKey && verifyCreators) ? 1 : 0,
```